[BUG] SerializeAttribute not valid on Event type with Unity 6.3

**Describe the bug**
Starting fron Unity 6000.3, it seems they changed the "SerializeField" attribute.
The error is :
Library\PackageCache\com.unity-atoms.unity-atoms-core@755a7bc71609\Runtime\Events\AtomEvent.cs(28,10): error CS0592: Attribute 'SerializeField' is not valid on this declaration type. It is only valid on 'field' declarations.

**Expected behavior**
No error message on opening a project with unity atoms.

**Environment**
 - Unity Version 6000.3.0b3
 - Version of Unity Atoms v4.5

--- a/Packages/Core/Runtime/Events/AtomEvent.cs
+++ b/Packages/Core/Runtime/Events/AtomEvent.cs
@@ -25,7 +25,6 @@ namespace UnityAtoms
 
         public int ReplayBufferSize { get => _replayBufferSize; set => _replayBufferSize = value; }
 
-        [SerializeField]
         protected event Action<T> _onEvent;
 
         /// <summary>

